integration_test/sql/migration.exs

```diff
@@ -65,6 +65,31 @@ defmodule Ecto.Integration.MigrationTest do
     end
   end
 
+  defmodule AlterForeignKeyMigration do
+    use Ecto.Migration
+
+    def up do
+      create table(:alter_fk_migration_users)
+
+      create table(:alter_fk_migration_posts) do
+        add :alter_fk_migration_user_id, :integer
+      end
+
+      alter table(:alter_fk_migration_posts) do
+        modify :alter_fk_migration_user_id, references(:alter_fk_migration_users, on_delete: :nilify_all)
```

Comment:
Just for sanity checking purpose, can you add another column at the same time you modify? Just to make sure the SQL syntax won't choke.